_pages/ato/levels.md

```diff
@@ -17,6 +19,24 @@ For more information, see [NIST 800-18](http://csrc.nist.gov/publications/nistpu
 
 Note that Open Data won't be listed, as it was developed within 18F.
 
+### Categorize
+
+* At 18F, the vast majority of use cases and data types have been categorized by the designated 18F AO, who is currently the Infrastructure Director (Noah Kunin).
+* Low, Moderate, High refer to the impact on either the government or the public if the metric is lost.
+     * Technically there is no overall level. For example, a high level for availability does not necessarily indicate high confidentiality or integrity.
+* The `Open Data` level is defined by confidentiality N/A, integrity low, and availability low
+* conditions
+    * [Sensitive PII](../../security/pii/) always raises the level to at least Moderate for the confidentiality and integrity metrics.
+    * If there is any authorization or authentication being done, it is likely at the Moderate level for all metrics.
+* The framework we usually use, since we must address a high water mark methodology anyway, is to ask ourselves (and the agency we are creating the system with) 3 worst case scenario questions:
+    * What is the worst possible outcome if all of the confidentiality of the system is lost?
+    * What is the worst possible outcome if all of the integrity of the system is lost?
+    * What is the worst possible outcome if all of the availability of the system is lost?
+* The answer to each question should then be interpreted in terms of impact to either the government or the public. The high water mark for either impacted party must be used.
+* The full list of data and functions in and of the system (in government parlance "mission based information types and delivery mechanisms") must be itemized in structured data. While the data types are obviously arbitrary and custom to each system we produce, the Government has a formalized data set of mission functions that should be mapped to the system via NIST 800-60.
+    * for a Rails app, this can simply be a link to the `db/schema.rb` on GitHub
```

Comment:
Good question – I have no idea! https://github.com/18F/before-you-ship/issues/126